### ISSUE TITLE
Add documentation on how to integrate sonarwhal into the development flow

### ISSUE THREAD

**Issue by molant** [sonarwhal/sonarwhal#1037]
There are a few ways to integrate sonarwhal during the development and I expect to have several "articles" ore recipes:

* [ ] Using the local connector while developing
* [ ] In the dev machine running a web server (such as `http-server` if static)
* [ ] In travis/Jenkins.
  *  This could be similar to the local server or integrate it with a staging environment and if it passes auto publish in production. I have an idea on how to do this with Azure but I'm not sure if we need to go as deep as to show how to do it on each Cloud provider.

**Comment by molant** [comment]
* [ ] Others?

Will start working on some of these and do separate PRs. If anyone wants to contribute (writing or proposing ideas) feel free!

**Comment by alrra** [comment]
Cc @schalkneethling

**Comment by schalkneethling** [comment]
Thanks for pining me on this @alrra

**Comment by molant** [comment]
@schalkneethling there's [a new PR](https://github.com/webhintio/hint/pull/1238) with steps on how to integrate Travis CI + Azure Web App + webhint. I guess other clouds have similar concepts.

Sorry it took so long but the renaming was longer than expected 😔

**Comment by ststimac** [comment]
@molant I assume this is still something that needs to be written? I'm adding things to the backlog for our Docs vendor.

**Comment by ststimac** [comment]
And tagging @antross in case he can answer my question above :)

**Comment by antross** [comment]
Yes, though we'll probably need to get a bit more specific to help. 

We've started work on additional CI documentation separate from this issue (which may itself benefit from some help if we don't finish it first), but if nothing else I know we have additional scenarios for using the [local connector](https://webhint.io/docs/user-guide/connectors/connector-local/) while developing that would be good to cover with examples (e.g. watch mode).
